website/src/app/[lang]/[region]/(website)/transparency/recipient-selection/[currency]/currency-redirect.tsx

```diff
@@ -0,0 +1,18 @@
+'use client';
```

Comment:
This file is the same as the one in the finances directory, right?

Let's move that file to `src/components` and re-use it?